src/npctalk.cpp

```diff
@@ -968,8 +968,13 @@ std::string dynamic_line(talk_topic topic, npc *p)
             }
 
         case TALK_SHELTER_PLANS:
-            return _("I don't know, look for supplies and other survivors I guess.");
-
+            switch (rng(1, 5)) {
+                case 1: return _("I don't know, look for supplies and other survivors I guess.");
+                case 2: return _("Maybe we should start boarding up this place.");
+                case 3: return _("I saw a helicopter wreck on my way here, maybe we can find some useful stuff outside.");
```

Comment:
I was thinking perhaps the npc is from a town far away. Generally, the player will probably find a helicopter wreck early on. (at least I always do). I can see how it creates false promises, though.